independent-projects/bootstrap/app-model/src/main/java/io/quarkus/bootstrap/model/AppModel.java

```diff
@@ -62,20 +62,14 @@
 
     private final Map<String, String> platformProperties;
 
-    private AppModel(AppArtifact appArtifact, List<AppDependency> runtimeDeps, List<AppDependency> deploymentDeps,
-            List<AppDependency> fullDeploymentDeps, Set<AppArtifactKey> parentFirstArtifacts,
-            Set<AppArtifactKey> runnerParentFirstArtifacts, Set<AppArtifactKey> lesserPriorityArtifacts,
-            Set<AppArtifactKey> localProjectArtifacts) {
-        this(appArtifact, runtimeDeps, deploymentDeps, fullDeploymentDeps, parentFirstArtifacts, runnerParentFirstArtifacts,
-                lesserPriorityArtifacts,
-                localProjectArtifacts, Collections.emptyMap());
-    }
+    private final Map<String, CapabilityContract> capabilitiesContracts;
 
     private AppModel(AppArtifact appArtifact, List<AppDependency> runtimeDeps, List<AppDependency> deploymentDeps,
```

Comment:
Is this constructor still used? It might be better to move everything over to the builder constructor.

Reply:
It isn't, so I removed it. Now I noticed this class implements `java.io.Serializable`. But it does not declare any non-private constructor. I guess I can also remove `Serializable` from it.